lldb/test/API/functionalities/fork/concurrent_vfork/main.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <thread>
+#include <unistd.h>
+#include <vector>
+
```

Comment:
Might be a good idea to create a global `std::vector<pid_t>` and a mutex to protect it. And any child processes that get forked get added to this vector. So maybe add:
```
std::mutex g_child_pids_mutex;
std::vector<pid_t> g_child_pids;
```
I will comment below where to use it.